arangod/Cache/CachedValue.cpp

```diff
@@ -60,6 +60,7 @@ CachedValue* CachedValue::construct(void const* k, std::size_t kSize,
       (reinterpret_cast<std::size_t>(buf) + _headerAllocOffset) & _headerAllocMask);
   std::size_t offset = buf - aligned;
   // ctor of CachedValue is noexcept
+  // cppcheck-suppress memleak
   return new (aligned) CachedValue(offset, k, kSize, v, vSize);
```

Comment:
This is superior to the changes I made in https://github.com/arangodb/arangodb/pull/13921. :+1: